action.yaml

```diff
@@ -305,17 +311,19 @@ runs:
         }
 
         failures <- x$get_solution()$failures
-        failed_pkgs <- failures[1, "failure_down"][[1]]
-        cli::cli_alert_warning("Failed to resolve dependencies. Failed packages: {.pkg {failed_pkgs}}.")
-        failures
+        failed_refs <- failures[1, "failure_down"][[1]]
+        cli::cli_alert_warning("Failed to resolve dependencies. Failed refs: {.pkg {failed_refs}}.")
+        print(failures)
 
-        if (identical(failures, failures_prev)) {
+        failure_message <- failures$failure_message
+        if (identical(failure_message, failures_message_prev)) {
```

Comment:
FYI: `failures` object is a pretty complex structure including desc objects (R6 class) which is always not identical. Hence changed into checking more simpler structures.